### PR TITLE
Add boss animations

### PR DESCRIPTION
**Pull request by NicolasLaure**

### CODE REVIEW

**Comment by NicolasLaure** [comment]
[[BOSS] Add laser attack animation](https://trello.com/c/L3FEWrC8/285-boss-add-laser-attack-animation)

**Comment by NicolasLaure** [comment]
[[IMPLEMENTATION] Boss's animations](https://trello.com/c/H6R9ZO4D/318-implementation-bosss-animations)

**Comment by NicolasLaure** [comment]
[[IMPLEMENTATION] Add incoming and escaping boss sequences](https://trello.com/c/KlDNLYa3/339-implementation-add-incoming-and-escaping-boss-sequences)

**Review by Maxi-F — APPROVED** [review]

File: Assets/Scripts/Enemy/Animation/EnemyAnimationHandler.cs [hunk 1]
```diff
@@ -42,4 +48,25 @@ public void SetLaserBlend(float value)
...
+    }
+
+    public void Recover()
+    {
+        _animator.SetTrigger(recover);
+    }
+
+    public void ReceiveHit()
+    {
+        if (_animator.GetCurrentAnimatorClipInfo(0)[0].clip.name != "hitVulnerable")
```
[the remark this answers is not in the document]
Comment (NicolasLaure) [reply Assets/Scripts/Enemy/Animation/EnemyAnimationHandler.cs:64]: Yup, Horrible

[the remark this answers is not in the document]
Comment (NicolasLaure) [reply Assets/Scripts/Enemy/Animation/EnemyAnimationHandler.cs:64]: We might check on thursday if there's a better way to resolve it

**Pull request merged**